app/controllers/api/collections_controller.rb

```diff
@@ -217,5 +219,22 @@ def collection_kml(collection, results)
     send_data sites_kml, type: 'application/vnd.google-earth.kml+xml', filename: "#{collection.name}_sites.kml"
   end
 
+  def check_user_member_admin!
```

Comment:
Code Duplication between check_user_member_admin! and check_user_member
For check_user_member_admin! = check_user_member && current_user.admin?